### DISCUSSION TITLE
Archived status missing from frontmatter (but there is both trashed and deleted)

### DISCUSSION THREAD

**Discussion by mbafford** [ndbeals/keep-exporter#11] [Ideas]
I don't see any indication of archive status making it to the frontmatter. I see `trashed` and `deleted`, but I don't have any notes with those set to true.

Assuming `trashed` precedes `deleted`.

When does `deleted` become *deleted*?

**Comment by mbafford** [comment]
Confirmed the order is `trashed` (goes in Google Keep's "Trash" label), then you delete the note, and it becomes deleted and isn't returned by the API. The `deleted` doesn't seem to get a value - at least not in my rapid test. 
So, unless the "deleted" frontmatter metadata is used to indicate a note no longer being returned by the gkeep API - then it may never be a useful value.

**Comment by ndbeals** [comment]
I'm pretty sure `deleted` only exists for the 7 (or 14) day period when you delete a note (rather than archiving which is trashed).

The `trashed` and `deleted` frontmatter headers are timestamps for when it was trashed/deleted.

**Reply by ndbeals** [reply]
So, yeah, I'll make a `status` header entry too.
